apps/checkout-app/.env.production

```diff
@@ -1 +0,0 @@
-NEXT_PUBLIC_VERCEL_URL=${VERCEL_URL}
```

Comment:
I see this env is still referenced in https://github.com/saleor/saleor-checkout/blob/main/apps/checkout-app/constants.ts#L16 but it does not seem to be used anywhere, please remove it from the envVars config as well as all references to it.